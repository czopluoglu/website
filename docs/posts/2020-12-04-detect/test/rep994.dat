10011000110110111111
10110111101111111000
00110101101000100110
10000010010100111010
10101101011100101111
01100101000001100011
10001101001110100101
00000011011000110010
01000000001001000000
01010001111101111101
01111110110110001100
11101011111100100110
00111001001011001101
11110111111111110111
01000000001010110101
10011111111011000111
10101101011111011111
10000000011101111000
10111001111111111111
01111101011111011011
10101101001110111111
10111111101110000010
00111000001001000110
10000101000010010011
10001011111100100010
00010000000100101100
11110111111000001111
11010111110111111111
00111000001000101111
11010100001001000000
01111101010111100101
10100111111011111110
11101100001110111111
10101111111011101111
11111111111111111100
11010100010010001110
00100001100010011110
00101000001000000111
01000101111000100111
11100011001011100111
00011010001111100110
11010101111001101110
00111111101101101101
00110101101111110110
11101101001110100101
01011001001111111100
10011110111110001111
00101101011000100011
10001101001101000011
10100000101001001111
11010100011111001110
11101001110001001000
11110111111111111110
00011000000100000111
11111101010111110011
11110110011110100010
11100001011101010010
10111101111000101111
10100110101111111111
11110000011001101010
11000000110011001111
11011101110000010110
00101111100101111110
11101111111111110111
10000101001001000010
01001001000101100011
11111101110110100111
11001100100101000100
01111000000000110011
10100111110100100000
10001101011100001010
00100001111111111111
00011110111010001110
10001100101001110011
00001001011010111011
01110010111010101000
11011111111010100111
11001110101010111110
11100101111010111110
10100100011011111011
11011001110011110111
00001010000000110000
11101101111111111111
00011001010110011011
11010001011100101000
10100000100110111111
10000101101011111110
10111110111000100110
11110101001100100111
00000101100111100110
00010100000001000000
01000101001010110111
00101000001110101110
00000000001010100110
10011101111001010110
00100111011111011110
11101000001010001100
11100001101101111111
11111001000010100010
11011100011010101111
00000001011110101001
11100001100101110011
11101001101000011110
10001110101000010011
10011111111001101010
11000100111111000111
00011111001100101111
11110100010100000011
10100011111110101111
10001011100111101111
00000000001110101010
00101001101000100110
11100101100100110110
10000110011001001110
01010100000000000000
01011101111010100010
11010001010011100010
10110000100010101000
10011111001100111001
10000101000111101110
00000001011010001010
10000100001111111110
11111110111110101111
11110011011111101111
10110011011101000110
01011111111110101111
00100000111111011111
01111111110110111110
11010111110101011111
00000000111100001110
11100010100010000010
00010100101110011110
00000000001000001001
10011010001010000111
10011101010111110100
01001101000100101111
00001101111110001101
11001111100010001001
00101111011101101101
10001100100000110010
11000010011101110111
01110101011001111111
10100001001010111111
11110001111101011111
00101001001111011111
10111101111110101111
00001111011000000111
00000001001100110111
11011101011010010011
00111001010101110011
10000000001011110111
10000100000000000000
11111111011110100101
10001001101010011110
11000011111110111110
10000101011110111111
10101101001110000011
00000100110010111010
01101010001000111010
11010100011010010001
11000101010010000110
10101111111110111111
00011001100010111010
00001001101110110111
10001001011011111011
01010000001001001110
00011101010011111111
11111001111000001111
11000101111010000100
00010001100001010010
00010100000110010011
10001011000010101111
10000000011011000111
00010010000111100111
00100001100011101011
10011010111010111111
10100000001000100010
01001010011101110111
11100101011001101100
11101001110110101110
00010101001111110110
10010111001101101011
10110101011011101111
00100111111110011110
00100101000110111001
10110001011110011110
10011111011110100101
11101001111010100010
00100000011010000110
11101000011010011010
01100100001010110011
10000110111011011011
10101100110110110010
01011000101000111011
11110011010110111111
00101100001010010001
10010000100011101011
10000101011111001110
00010101111011001001
10001100001111101110
00000000101000100110
00000111111010010010
11011111111101111111
10000111111110111110
10000100000001000010
00100000000101100011
00010000001011001111
00111111011101111101
11110011110010000000
10101001101010110011
00000100011010110010
00000001001101001110
01001001001010100111
11011000000011111011
00000010011011111011
11000101001101111011
10001111011010100011
11000001000010100110
01100010111011101110
11101011111111100111
11010011100100100111
00101111111111110111
10110100001101001111
00110001101111111011
00100110001110111011
11010110110000111111
00101101011110100100
11010111100110010110
10000001111000010000
00001010001010010100
11110111100000101100
01001001100000000110
11101111011110101101
11100111111110101010
01010111111110011110
10011111111001010101
01010111110011110110
10110111111011111110
00101101011000000110
00111100011010101111
00101111101000001111
11111001011101000100
00001101111000101111
00000101011111101111
01100101101010001010
11111101010010101111
10100000101010010110
10111111111101001111
01010000000111100110
10111001001111100111
