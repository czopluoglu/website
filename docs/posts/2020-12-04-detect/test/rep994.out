-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep994.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.80

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.60
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7635
                   IDN index value:   0.6895
                           Ratio r:   0.5952

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    1    2    1    1    2    1    1
    3    2    4    3    2    4    3    3    4    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    4    6    7    9   10

   -----------CLUSTER  2-------------

    3    5    8   12   15   20

   -----------CLUSTER  3-------------

   11   14   17   18

   -----------CLUSTER  4-------------

   13   16   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+-++++-------+----+
   +d+--+++-+-+----+---
   -+d++-+-+-----+-+---
   +-+d+++-+++--+---+--
   +-++d++-+-++-+--+---
   ++-++d+--+-----++-++
   ++++++d--+---+-+-+--
   -+-----d++++-+-+----
   --+++--+d+-+++--+---
   -+-+-++++d++--+-+---
   ---++--+-+d++-+++-+-
   -+--+--++++d+----+++
   --------+-++d++---++
   ---++-+++---+d+++--+
   +-+------++-++d++-++
   -----+++--+--++d+++-
   -++-++--+++--+++d+-+
   ---+--+----+---++d++
   -----+----+++-++-+d+
   +----+-----++++-+++d
 
 No cross validation for this DETECT run
